zeroX:  200.0
zeroY:  0.0
zeroZ:  -35
zeroR:  0.0
